curriculum/challenges/english/06-information-security-and-quality-assurance/quality-assurance-and-testing-with-chai/use-regular-expressions-to-test-a-string.english.md

```diff
@@ -7,8 +7,7 @@ challengeType: 2
 ## Description
 <section id='description'>
 As a reminder, this project is being built upon the following starter project on <a href='https://glitch.com/#!/import/github/freeCodeCamp/boilerplate-mochachai/'>Glitch</a>, or cloned from <a href='https://github.com/freeCodeCamp/boilerplate-mochachai/'>GitHub</a>.
-#match Asserts that the actual value
-matches the second argument regular expression.
+<code>#match</code> asserts that the actual value matches the second argument regular expression.
```

Comment:
This is not an instruction.

Reply:
It is not a part of instruction in my commit @RandellDawson